ws2812b/inc/ws2812b_conf.h

```diff
@@ -0,0 +1,45 @@
+/** 
+  ******************************************************************************
+  * @file    ws2812b_conf.h
+  * @brief   configuration file
+  * @author  https://github.com/j3qq4hch
+  * @author  https://github.com/mnemocron
+  * @date    2020-10-27
+  * 
+  * @note    IAR Embedded Workbench 3.11.1
+  *          shared components 8.3.2.5988
+  * @note    tested with RGBW LED type: IN-PI55QATPRPGPBPW-XX
+  * 
+  ******************************************************************************
+  */
+
+#ifndef WS2812B_PIN_H
+#define WS2812B_PIN_H
+
+/* ---------------------------------------------------------------------------*/
+/* GPIO Port Register Map */
+/** @todo why not use SPL definitions? */

```

Comment:
SPL is STM library, there are other toolchains as well. It's better to not have dependencies on toolchains, and if you have them, place to separate file